adminpages/orders.php

```diff
@@ -172,6 +172,31 @@
 	}
 }
 
+// Refund this order
+if ( ! empty( $_REQUEST['refund'] ) ) {
+	// Check nonce for refunding.
+	$nonceokay = true;
+	if ( empty( $_REQUEST['pmpro_orders_nonce'] ) || ! check_admin_referer( 'refund_order', 'pmpro_orders_nonce' ) ) {
+		$nonceokay = false;
+	}
+
+	$rorder = new MemberOrder( intval( $_REQUEST['refund'] ) );
```

Comment:
`(int)` cast is faster https://hakre.wordpress.com/2010/05/13/php-casting-vs-intval/

```suggestion
	$rorder = new MemberOrder( (int) $_REQUEST['refund'] );
```